lib/embed/request.rb

```diff
@@ -70,7 +70,7 @@ def as_url_params
         :hide_title, :hide_embed, :hide_search, :hide_download,
         :min_files_to_search,
         :canvas_id, :canvas_index,
-        :search, :suggested_search
+        :search, :image_tools
```

Comment:
It looks like `suggested_search` was dropped here.

Reply:
Just.. uh.. testing you.